back/cmd/openbsrv/main.go

```diff
@@ -28,6 +39,7 @@ func run() error {
 		migrate   bool
 		rollback  bool
 		skipsrv   bool
+		debug     bool
```

Comment:
Debug logger should be a project-wide-global, not just a main package-wise global.
- callable from anywhere in the program
- not racy 
- can be turned on and off even in the middle of the program

Reply:
I believe I have made the above changes in this branch.

I have come up with two solutions for ensuring debug logging is thread-safe. The first is in this branch and it uses the sync/atomic package. The second is in the [logging-policy-debug-mutex-version branch ](https://github.com/OpenEugene/openboard/tree/logging-policy-debug-mutex-version)and it uses the sync.Mutex type.

I'd first like to see what @daved thinks about these two implementations before I create benchmark tests for them. Whichever does better in benchmarking will be the one I'll submit for the final PR.